src/components/Create/Create.js

```diff
@@ -179,14 +258,65 @@ const Create = ({ history }) => {
                     required
                   />
                 </Form.Item>
-                <Form.Item>
-                  <Input
-                    placeholder="white paper/document hash"
-                    name="documentHash"
-                    onChange={e => handleChange(e)}
-                    required
-                  />
+                <Form.Item style={{ lineHeight: "0px" }}>
+                  <Dragger
+                    multiple={false}
+                    action="https://www.mocky.io/v2/5cc8019d300000980a055e76"
+                    transformFile={transformFile}
+                    beforeUpload={beforeUpload}
+                    onChange={handleChangeUpload}
+                    onRemove={() => false}
+                    fileList={fileList}
+                    name="documentHashUpload"
+                    style={{
+                      background: "#D3D3D3",
+                      borderRadius: "8px"
+                    }}
+                  >
+                    <Icon style={{ fontSize: "24px" }} type="upload" />
+                    <p>Click or drag file to this area to upload (optional)</p>
```

Comment:
Change text to remove `upload` -- the file is not being uploaded and we want it to be clear that the file itself is not submitted to any server

Text should read:
Top: `Click, or drag file to this area to hash.`
Bottom: `The hash is performed client-side and the file is not uploaded.`

Reply:
Remove `(optional)` text -- we should instead clearly identify required fields with a `*` and handle any Create Token submission missing required fields in form validation.